webhooks.py

```diff
@@ -121,7 +122,7 @@ def index():
 
         elif event in ['push']:
             # Push events provide a full Git ref in 'ref' and not a 'ref_type'.
-            branch = payload['ref'].split('/')[2]
+            branch = payload['ref'].split('/', 2)[2]
```

Comment:
We have repos called "feature/this_and_that" which are perfectly fine and used by gitflow approach. the split you proposed is doing:

ref/master/feature/this_and_that -> causing that "feature" is selected as branch name, rather than "feature/this_and_that"